src/main/java/org/ibp/api/domain/user/UserProfileDto.java

```diff
@@ -0,0 +1,80 @@
+package org.ibp.api.domain.user;
+
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+@AutoProperty
+public class UserProfileDto {
+
+	private String userName;
```

Comment:
I would not send the userName, we are changing the profile for the logged in user, so we can get the user from the session, get the username from there and validate the password